### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by NihitYadav**
Edit the PR description here.

### CODE REVIEW

**Comment by PatelNeelMahesh** [comment]
You should give better commit message, instead of `frontend done`, 
You can refer given link here: https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/

File: lvx frontend/src/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```
Comment (PatelNeelMahesh) [inline lvx frontend/src/setupTests.js]: You can delete the file if you are not writing testcases.

**Review by PatelNeelMahesh — APPROVED** [review]
Make the required changes before merging pull request.

File: lvx frontend/src/setupTests.js (hunk 1 above)
Comment (NihitYadav) [reply lvx frontend/src/setupTests.js]: Deleted this file

**Pull request merged**